airbyte-integrations/connectors/source-intercom/manifest.yaml

```diff
@@ -89,40 +91,27 @@ definitions:
             class_name: source_declarative_manifest.components.ErrorHandlerWithRateLimiter
             response_filters:
               - type: HttpResponseFilter
+                http_codes:
+                  - 401
                 action: FAIL
                 failure_type: config_error
                 error_message: >-
-                  Failed to perform request. Error: Active subscription needed.
-                  Please, validate your current Intercom plan to continue using
-                  API.
-                error_message_contains: Active subscription needed
+                  Failed to perform request. Error: Permission Scope Error.
+                  Please validate you have "Read admins" permission here: https://app.intercom.com/a/apps/_/developer-hub
         record_selector:
           type: RecordSelector
           extractor:
             type: DpathExtractor
             field_path:
               - admins
-        paginator:
```

Comment:
just to confirm, none of these endpoints use pagination anymore? The docs for these seem to indicate that there is no longer pagination and all records are returned, but it just seems surprising to see the feature removed.

But if thats what it looks like in the response, then fine by me